misc/userscripts/readability

```diff
@@ -21,20 +21,35 @@ tmpfile = os.path.join(
 if not os.path.exists(os.path.dirname(tmpfile)):
     os.makedirs(os.path.dirname(tmpfile))
 
+# styling for window size depended margin scaling and line height
+header = """<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>%s</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <style type="text/css">
+       body{margin:40px auto;max-width:650px;line-height:1.4;padding:0 10px}
+       h1,h2,h3{line-height:1.2}
+    </style>
+</head>
+"""
+
 with codecs.open(os.environ['QUTE_HTML'], 'r', 'utf-8') as source:
     data = source.read()
 
     try:
         from breadability.readable import Article as reader
         doc = reader(data)
-        content = doc.readable
+        title = doc._original_document.title
```

Comment:
Is there a way you can get that information without needing to access private properties? Like this, it's possible that things break even with minor breadability updates as we're using internal API.

Reply:
Not using functions exposed by the module, no. The only alternative I can think of is parsing the html manually. However, inspecting the [commits for breadability](https://github.com/bookieio/breadability/commits/master), shows that no changes to the code base (i.e. non test and documentation related changes) have been done since 2014(!) - the project is basically unmaintained. Hence, the possibility that this particular private property  will change is very slim. So I suggest leaving it as is, finding another readability module, or relying purely on `python-readability`.